Client/Frontend/Strings.swift

```diff
@@ -5307,7 +5307,7 @@ extension String {
     public static let LoginsListNoLoginsFoundDescription = MZLocalizedString(
         key: "LoginsList.NoLoginsFound.Description.v119",
         tableName: nil,
-        value: "Saved logins will show up here. If you saved your logins to Firefox on a different device, sign in to your account.",
+        value: "Saved logins will show up here. If you saved your logins to %@ on a different device, sign in to your account.",
```

Comment:
Please explain what the placeholder will be replaced with (in this case, Firefox).